.circleci/config.yml

```
@@ -224,9 +224,17 @@ jobs:
                 git checkout --force -B "$CIRCLE_BRANCH" "$CIRCLE_SHA1"
                 git --no-pager log --no-color -n 1 --format='HEAD is now at %h %s'
             fi
+      - run: cat package*.json > cachekey
+      - restore_cache:
+          keys:
+            - v7-root-npm-{{ checksum "cachekey" }}
```

Est-ce que ca ne serait pas suffisant?
```suggestion
            - v7-root-npm-{{ checksum "package-lock.json" }}
```

---

J'avoue ne pas avoir spécialement réfléchi et ait copié ce qu'on fait sur les autres caches.

Il me semble que certaines modifs dans le `package.json` ne nécessitent pas nécessairement de changement dans le `package-lock.json`, par exemple un nouveau script qu'on exécuterai dans la CI ? Je cherche une raison 😛